src/components/inspectors/CycleManager.js

```diff
@@ -2,69 +2,69 @@ import { DateTime } from 'luxon';
 import { getIso8601FormattedDateString, getPeriod } from './TimeUtils';
 
 export default class CycleManager {
-  _startDate;
-  _repeat;
-  _periodicityValue;
-  _selectedWeekdays;
-  _endDate;
-  _ends;
-  _times;
+  #startDate;
+  #repeat;
+  #periodicityValue;
+  #selectedWeekdays;
+  #endDate;
+  #ends;
+  #times;
 
   constructor(startDate, repeat, periodicityValue, selectedWeekdays = [], endDate = null, ends = 'never', times = '1') {
-    this._startDate = this.sanitizeDate(startDate);
-    this._repeat = repeat;
-    this._periodicityValue = periodicityValue;
-    this._selectedWeekdays = selectedWeekdays;
-    this._endDate = this.sanitizeDate(endDate);
-    this._ends = ends;
-    this._times = times;
+    this.#startDate = this.sanitizeDate(startDate);
+    this.#repeat = repeat;
+    this.#periodicityValue = periodicityValue;
+    this.#selectedWeekdays = selectedWeekdays;
+    this.#endDate = this.sanitizeDate(endDate);
+    this.#ends = ends;
+    this.#times = times;
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set startDate(value) {
-    this._startDate = this.sanitizeDate(value);
+    this.#startDate = this.sanitizeDate(value);
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set endDate(value) {
-    this._endDate = this.sanitizeDate(value);
+    this.#endDate = this.sanitizeDate(value);
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set ends(value) {
-    this._ends = value;
+    this.#ends = value;
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set times(value) {
-    this._times = value;
+    this.#times = value;
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set repeat(value) {
-    this._repeat = value;
+    this.#repeat = value;
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set periodicityValue(value) {
-    this._periodicityValue = value;
+    this.#periodicityValue = value;
   }
 
   // no coverage needed for mutator
   // istanbul ignore next
   set selectedWeekdays(value) {
-    this._selectedWeekdays = value;
+    this.#selectedWeekdays = value;
   }
 
   /**
    * For historic reasons we need to be liberal in how we accept various incoming
-   * dates. See https://en.wikipedia.org/wiki/Robustness_principle
+   * dates. See https://en.wikipedia.org/wiki/Robustness#principle
```

Comment:
Was this change intentional? Generally, hashes / fragments like this will scroll the page to the element with that ID (`principle` in this case). But the `/wiki/Robustness` page has no such element. Meanwhile, the `/wiki/Robustness_principle` page is more relevant.

Reply:
Nah I don't think that is intentional nor do I think we really need the comment, this was a placeholder for a discussion at some point :)